### ISSUE TITLE
How to mask (ignore) first several outputs of an RNN?

### ISSUE THREAD

**Issue by MaratZakirov** [fchollet/keras#6365]
Suppose I have char-rnn which I want RNN to learn how to predict next character given 120 previous characters. The problem is that I do not know how to ignore several first (10-20) characters which of course produces huge error.

```
model.add(LSTM(128,  return_sequences=True, input_shape=(120, len(chars))))
model.add(TimeDistributed(Dense(len(chars), activation='softmax')))
model.compile(loss='categorical_crossentropy', optimizer=RMSprop(lr=0.01))
```

How to mask only output? Make custom loss function?
Or is it enough to me (for categorial cross entropy loss) is just setting first output characters as full zero vectors?
